content/en/blog/2023/lambda-release.md

```diff
@@ -0,0 +1,48 @@
+---
+title: OpenTelemetry Updates our Lambda Support
+linkTitle: OTel Lambda Updates
+date: 2023-05-15
+author: '[Carter Socha](https://github.com/cartersocha) (Lightstep)'
+---
+
+The Functions-as-a-Service (FAAS) SIG is incredibly excited to announce that the
+OpenTelemetry (OTel) community is now releasing our own Lambda layers and has
+[centralized documentation](https://opentelemetry.io/docs/faas/) on how to
+monitor Amazon Web Services (AWS) Lambdas.
+
+### A Slightly New World
+
+If you've been monitoring Lambdas using OTel for a while now you may be slightly
+confused by this announcement. You might say something like OTel has had a
+[repo for Lambda layers](https://github.com/open-telemetry/opentelemetry-lambda)
+and they've been available on AWS for years.
+
+Which is totally correct. Thankfully, we're not remaking the wheel here.
+However, there are a couple new updates.
+
+Previously the OTel Lambda layers were only released as part of the
```

Comment:
I would split the things in this paragraph into a list to make it more clear what's changing and why

Reply:
Updated and lmk what you think.